src/plugins/projectexplorer/jsonwizard/jsonwizardfactory.cpp

```diff
@@ -611,7 +611,7 @@ bool JsonWizardFactory::initialize(const QVariantMap &data, const QDir &baseDir,
         *errorMessage = tr("No displayCategory set.");
         return false;
     }
-    setDisplayCategory(strVal);
+    setDisplayCategory(Utils::globalMacroExpander()->expand(strVal));
```

Comment:
I think we should do the same for displayName and description at least.

Reply:
Done inside PR: https://github.com/sailfishos/sailfish-qtcreator/pull/264